src/test/unit/controls/Line/LineLoad-spec.js

```diff
@@ -349,6 +349,24 @@ describe("Line - Load", () => {
                 input.key
             );
         });
+        it("adds line with correct stroke-dasharray", () => {
+            graphDefault.destroy();
+            graphDefault = new Graph(getAxes(axisDefault));
+            input = getInput(valuesDefault, false, false);
+            input.style = {
+                strokeDashArray: "2,2"
```

Comment:
This is great but can we also have test for space separated ones?
Test 1: `2,2`
Test 2: `2 2`
Test 3: `2`

Notice that all these yield the same output.